moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -648,6 +772,19 @@ void IKFastKinematicsPlugin::getSolution(const IkSolutionList<IkReal>& solutions
   }
 }
 
+double IKFastKinematicsPlugin::respectLimits(double joint_val, double min, double max) const
+{
+  while (joint_val > max && joint_val - 2 * M_PI > min)
```

Comment:
add comment explaining this math logic

was this functionality not already available??

Reply:
Amazingly, no